### PR TITLE
[#10433] Instructor view results: Use fixed-width floating numbers

### LINKED ISSUE #10433: Instructor view results: Use fixed-width floating numbers
V7:
image

V6:
image

### PR DESCRIPTION
**Pull request by swood049**
Fixes #10433

I added a DecimalPipe object inside of mcq-question-statistics.componenets.ts, and I used it to format the information given to the table as requested in the issue (with 3 significant digits). It changes only the Average and Total columns, as well as the weight numbers in the header.

### CODE REVIEW

File: e [hunk 1]
```diff
@@ -0,0 +1,388 @@
+[33mcommit 1c2b94d63a7be533d96e149223c0f4368bcea1cd[m[33m ([m[1;36mHEAD -> [m[1;32mmaster[m[33m, [m[1;31mupstream/master[m[33m)[m
```
Comment (madanalogy) [inline e:1]: What is this file? Could you remove it from your PR?

**Review by madanalogy — CHANGES_REQUESTED** [review]
Your PR seems to be failing the lint check, please execute `npm run lint` and fix the errors that pop up

File: e (hunk 1 above)
Comment (swood049) [reply e:1]: I believe that this is referring to the module "DecimalPipe" module that I am using to make the change that is asked for. I am not 100% certain on how to use the module without importing it.

Comment (swood049) [reply e:1]: Upon further review, I have no idea what this file is, and I do not think that I made that change.

**Comment by madanalogy** [comment]
Could you please add some screenshots as well to verify the fix? Thanks!

File: e (hunk 1 above)
Comment (swood049) [reply e]: I made the changes requested.

**Review by swood049 — COMMENTED** [review]
I made the requested changes.

**Comment by madanalogy** [comment]
> I made the requested changes.

Please see my comment above regarding the addition of screenshots

**Comment by swood049** [comment]
The first photo below shows that I deleted the file, and that I moved the location of the import statement (it also says the lint test passes). 
image
I just tested it in my local development. Everything seems to be working as requested.
Proof that I made the change:
image

The local lint test: 
image

Check to see that it runs:
image

**Comment by madanalogy** [comment]
> I think that these are what you need. If I am wrong, let me know.

No that is not what I meant sorry. Refer to the original issue filed, there are screenshots showing the numbers compared between v6 and v7 https://github.com/TEAMMATES/teammates/issues/10433#issue-665729617

What you need to do is replicate the process (i.e. create a feedback session, then submit dummy statistics) and take a screenshot showing your changes. This screenshot should show the same UI component as the original issue filed.

**Comment by swood049** [comment]
Ah! Yes! Got it!
I took the liberty to do the same with the column titles.
image

File: src/web/app/components/question-types/question-statistics/mcq-question-statistics.component.ts [hunk 2]
```diff
@@ -62,7 +66,7 @@ export class McqQuestionStatisticsComponent extends McqQuestionStatisticsCalcula
       { header: 'Recipient Name', sortBy: SortBy.MCQ_RECIPIENT_NAME },
       ...Object.keys(this.weightPerOption).map((key: string) => {
         return {
-          header: `${key} [${this.weightPerOption[key]}]`,
+          header: `${key} [${this.changeSignificantDigits.transform(this.weightPerOption[key], '1.2-3')}]`,
```
Comment (madanalogy) [inline src/web/app/components/question-types/question-statistics/mcq-question-statistics.component.ts:69]: I'm not keen on the use of this string variable in-line, feels too much like a magic number. Could you maybe define a constant variable with a descriptive name? Preferably in the scope of this method.

**Review by madanalogy — CHANGES_REQUESTED** [review]

**Comment by madanalogy** [comment]
Closing due to inactivity

**Pull request closed without merging**